### DISCUSSION TITLE
Trigger Element ButtonAction soll nur false setzen können

### DISCUSSION THREAD

**Discussion by manoloxxl** [Zefau/ioBroker.jarvis#1161] [Q&A] [answered]
Hallo zusammen,

ich bin neu dabei und bin total begeistert von Jarvis. Bisher konnte ich fast alles selber bauen oder via Google Hilfestellungen finden. Nun stehe ich aber vor einem kleinen Problem und möchte gerne wissen, ob ich den Umweg über ein Script gehen muss, oder ob es direkt in Jarvis geht.

Ich habe einen Datenpunkt "All On" aus meinem deconz, der true / false gestellt werden kann.

True > alle Lampen gehen an
False > alle Lampen gehen aus

Nun möchte ich in Jarvis aber ausschließlich einen "alles aus" anbieten. 

Als Triggerelemt kann man wunderbar einen "Ausschalten" button bauen. Aber ich bekomme es einfach nicht hin, dass dieser beim klicken des Buttons IMMER false in den Datenpunkt schreibt. Er schreibt leider immer true.

Kann mir jemand verraten wie es geht? Ansonsten würde ich halt ein kleines Script erstellen...

Grüße

BTW: Kann man eigentlich den "Konfigurationsbutton" ausblenden? Ich möchte nicht, dass meine Gäste mir in der Config rumfummeln...

**Comment by Zefau** [comment]
Leider nicht möglich. Leg gerne einen Feature Request an.

Ausblenden des Konfigurationsbutton mit v3 einfach im Layout.

**Comment by manoloxxl** [comment]
Besten Dank!

**Comment by Bernd9000** [comment, marked as answer]
Also soviel ich weiß geht das.
Z.B. mit dem IconButtonAction und dann folgendes unter Datenpunkt Eigenschaften (JSON Format) eintragen:
![1](https://user-images.githubusercontent.com/77005685/141002030-0a231679-be94-441e-921a-bb595c6a8d72.jpg)

**Reply by Zefau** [reply]
Das stimmt, in den einzelnen Geräten geht das. Auch mittels `setOn` / `setOff` als weitere Alternative. Aber mittels Button im Layout über mehrere Geräte hinweg geht das nicht (bzw. nur, wenn jedes Gerät so konfiguriert ist wie von dir dargestellt).

Sprich es müsste eine Option im Button geben, dass immer ein bestimmter Wert gesetzt werden kann, unabhängig vom Gerät.

**Comment by Bernd9000** [comment]
Kann sein das ich es falsch verstanden habe.
Ich hab den TE so verstanden das Er schon ein Datenpunkt hat der alle Geräte ausschaltet und Er braucht nur noch ein Schalter der immer false liefert.

**Reply by manoloxxl** [reply]
Du hast mich richtig verstanden. Deine Lösung klappt wunderbar.

**Reply by RealRelaxo** [reply]
Ich habe dies nun ganz genau so konfiguriert.
- Eigener Datenpunkt nur für den AusschalteEvent
- json bei den Datenpunkt Eigenschaften: {"ON":"false","OFF":"false"}
- StateList Widget mit IconButtonAction
aber das Widget liefert mir beim Einschalten immer noch true.

Wo ist mein Fehler?
![widget](https://user-images.githubusercontent.com/112799002/188303496-ac704389-cfba-4bae-8011-c5c8a0983f75.png)
![device](https://user-images.githubusercontent.com/112799002/188303498-893d7a71-3fe5-4b91-8b2a-f09d9544345f.png)

**Reply by Zefau** [reply]
Versuch mal `{"on":"false","off":"false"}` statt `{"ON":"false","OFF":"false"}`

**Reply by RealRelaxo** [reply]
Das war es.
Wieder was gelernt,
Vielen Dank.
